src/Cake.Web/App_Data/blog/2016-01-18-cake-v0.8.0-released.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cake v0.8.0 released
+category: Release Notes
+---
+
+Version 0.8.0 of Cake has been released.
+
+In this release we've added support for making the `#tool` and `#addin` directives independent from a specific package manager, as well as adding support for fetching specific and prerelease versions of NuGet packages.
```

Comment:
I think it would make sense to provide a link to the soon to be released documentation around the directives.